src/components/v5/common/ActionSidebar/partials/forms/ManagePermissionsForm/ManagePermissionsForm.tsx

```diff
@@ -1,5 +1,8 @@
+import { Id } from '@colony/colony-js';
 import React, { FC, useCallback } from 'react';
+import { useWatch } from 'react-hook-form';
 
+import { USER_ROLE } from '~constants/permissions';
```

Comment:
Sorry to be a pain, but would you mind changing this to an enum if it's easy enough? It's one of those `as const` hacky type. No problem if not - can always open a maintenance issue for it.

Reply:
Oops, sorry man! I saw this comment after merging. I'll create the issue for this 👍

Reply:
There you go: https://github.com/JoinColony/colonyCDapp/issues/1739